src/util/extract-app-permissions-from-jwt.ts

```diff
@@ -0,0 +1,9 @@
+import * as jose from "jose";
+
+import { AppPermission } from "../../dist/types";
+
+export const extractAppPermissionsFromJwt = (jwtToken: string): AppPermission[] => {
+  const tokenDecoded = jose.decodeJwt(jwtToken);
+
+  return tokenDecoded.permissions as AppPermission[];
```

Comment:
**question**: Is there no need to parse it? Can we safely cast the type?

Reply:
Saleor guarantees that, if Saleor can't do that right, app will break anyway